### PR TITLE
Update article flow

### PR DESCRIPTION
**Pull request by abdullahcalisir12**
This Pr is related to new style of article add page.
I updated existing pages.
My mainly changes are on Articleadd.vue. I removed StepperManager and related functions from it.
I removed preview from ArticleAddStep3.vue.
I updated BaseDropdown.vue style because options modal were overlapping with dropdown itself

https://github.com/whynotearth/Volkswagen-email-corporate-communications-CMS/issues/242

### CODE REVIEW

File: src/components/ArticleAddStep2.vue [hunk 1]
```diff
@@ -75,24 +92,33 @@ export default {
...
+        required: requiredIf(context => {
+          return context.isFieldVisible('headline');
+        }),
+        maxLength: maxLength(80)
+      },
+      description: {
+        required: requiredIf(context => {
+          return context.isFieldVisible('description');
+        }),
+        maxLength: maxLength(this.isImagesEmpty ? 625 : 450)
```
Comment (mort3za) [inline src/components/ArticleAddStep2.vue:110]: Shouldn't be this 750 characters instead of 625?

Comment (paulchrisluke) [reply src/components/ArticleAddStep2.vue:110]: yes thats my fault in writing the issue. We can increase it to 750 after the meeting @mort3za

File: src/components/BaseDropdown.vue [hunk 2]
```diff
@@ -105,4 +115,12 @@ export default {
 .active {
   background: rgba(3, 179, 249, 0.12);
 }
+
+.dropdown-top-auto {
+  @apply top-auto !important;
+}
+
+.dropdown-border-0 {
+  @apply border-b-0 !important;
```
Comment (mort3za) [inline src/components/BaseDropdown.vue:124]: Is there any reason for using important? why not just increase the specificity by duplicating classname? like: `.dropdown-border-0.dropdown-border-0`

Comment (abdullahcalisir12) [reply src/components/BaseDropdown.vue:124]: yes there is. I didn't want to effect all places where baseDropdown used. So I created props which takes class names as string by default empty string and wanted to have this classes to overwrite existing classes which both effect same styles. But the first one gets priority not this one.

File: src/components/ArticleAddStep2.vue (hunk 1 above)
Comment (abdullahcalisir12) [reply src/components/ArticleAddStep2.vue:110]: updated here. https://github.com/whynotearth/Volkswagen-email-corporate-communications-CMS/pull/337

**Pull request merged**